pkgs/development/haskell-modules/configuration-common.nix

```
@@ -428,6 +428,12 @@ self: super: {
       # (this should be possible by also taking PREFIX into account).
       ./patches/git-annex-no-usr-prefix.patch
     ];
+
+    postPatch = ''
+      substituteInPlace Makefile \
+        --replace-fail 'InstallDesktopFile $(PREFIX)/bin/git-annex' \
+                       'InstallDesktopFile $(DESTDIR)$(PREFIX)/bin/git-annex'
```

Desktop files should not contain absolute paths.

```suggestion
                       'InstallDesktopFile git-annex'
```